packages/react-app/src/components/pages/Bounties/Bounty/index.tsx

```diff
@@ -223,26 +220,43 @@ const BountyModal = ({
   onClose: () => void;
 }): JSX.Element => {
 	// TODO Need a Delete action if Draft or Open
-
+	const { colorMode } = useColorMode();
 	return (
-		<Modal isOpen={isOpen} onClose={onClose}>
-			<ModalOverlay />
-			<ModalContent maxW={'700px'} borderWidth={3}>
-				<ModalHeader
-					bg={useColorModeValue('gray.200', 'gray.600')}
-					roundedTop="md"
-				>
-					<BountyHeader bounty={bounty} />
-				</ModalHeader>
-				<ModalCloseButton />
-				<ModalBody>
-					<BountyDetails bounty={bounty} />
-				</ModalBody>
-				<ModalFooter>
-					<BountyActions bounty={bounty} onCancel={onClose} />
-				</ModalFooter>
-			</ModalContent>
-		</Modal>
+		
+		<Center>
+			<Modal isOpen={isOpen} onClose={onClose}>
+				<ModalOverlay />
+				<ModalContent maxW={'700px'} borderWidth={3}>
+					<ModalHeader
+						bg={useColorModeValue('gray.200', 'gray.600')}
+						roundedTop="md"
+					>
+						<BountyHeader bounty={bounty} />
+					</ModalHeader>
+					<ModalCloseButton />
+					<ModalBody>
+						<BountyDetails bounty={bounty} />
+					</ModalBody>
+					<ModalFooter>
+						<Stack direction='row' spacing={1} alignItems='center' alignContent='center' verticalAlign={'center'}>
+							<BountyActions bounty={bounty} onCancel={onClose} />
+							 <Button boxShadow={'md'}
+								transition="background 100ms linear"
+								aria-label="Delete-button"
+								bg={colorMode === 'light' ? 'primary.300' : 'primary.700'}
+								size='md'
+								width='200px'
+								isLoading
+								loadingText='Delete this bounty'
+							>Delete this bounty</Button>
+							<BountyEditButton bounty={bounty} />
```

Comment:
This creates a duplicate Edit button:

![image](https://user-images.githubusercontent.com/2557404/184456203-e004f17e-351d-4867-a02a-504a8374289f.png)